index.js

```diff
@@ -48,10 +48,14 @@ function PgSimplifyInflectorPlugin(
     );
   }
 
-  const connectionSuffix = pgOmitListSuffix ? "-connection" : "";
-  const ConnectionSuffix = pgOmitListSuffix ? "Connection" : "";
-  const listSuffix = pgOmitListSuffix ? "" : "-list";
-  const ListSuffix = pgOmitListSuffix ? "" : "List";
+  const connectionSuffix = (omitListSuffixTag) =>
+    pgOmitListSuffix || omitListSuffixTag ? "-connection" : "";
+  const ConnectionSuffix = (omitListSuffixTag) =>
+    pgOmitListSuffix || omitListSuffixTag ? "Connection" : "";
+  const listSuffix = (omitListSuffixTag) =>
+    gOmitListSuffix || omitListSuffixTag ? "" : "-list";
+  const ListSuffix = (omitListSuffixTag) =>
+    pgOmitListSuffix || omitListSuffixTag ? "" : "List";
```

Comment:
If `omitListSuffix` is set to `0` we should make it unset `pgOmitListSuffix`. So `@omitListSuffix` -> true; `@omitListSuffix 0` -> false. Then if the tag is set, it should rule; otherwise pgOmitListSuffix is used as the fallback.

```suggestion
  function o(tag) { return tag != null ? tag !== "0" : !!pgOmitListSuffix; }
  const connectionSuffix = (omitListSuffixTag) =>
    o(omitListSuffixTag) ? "-connection" : "";
  const ConnectionSuffix = (omitListSuffixTag) =>
    o(omitListSuffixTag) ? "Connection" : "";
  const listSuffix = (omitListSuffixTag) =>
    o(omitListSuffixTag) ? "" : "-list";
  const ListSuffix = (omitListSuffixTag) =>
    o(omitListSuffixTag) ? "" : "List";
```

Reply:
Alternatively we could use `@omitListSuffix` / `@includeListSuffix` tags, but that's kinda awkward.

Actually, maybe `@listSuffix include/omit` would be better...?


```suggestion
  function o(tag) { return tag != null ? tag === "omit" : !!pgOmitListSuffix; }
  const connectionSuffix = (tag) =>
    o(tag) ? "-connection" : "";
  const ConnectionSuffix = (tag) =>
    o(tag) ? "Connection" : "";
  const listSuffix = (tag) =>
    o(tag) ? "" : "-list";
  const ListSuffix = (tag) =>
    o(tag) ? "" : "List";
```

Reply:
My thinking would be that it would be strictly additive - there's always a workaround to set to to false globally then enable it everywhere you want it.  Incidentally, I leave it off globally, otherwise all my connections (the vast majority of things) would have `Connection` suffixes.

Dumb question: with the restrictions on valid tag inputs, I assume you mean a string `0` there?  I would probably prefer some words there, and would also prefer to use a single tag if feasible.  But I agree the UX is better if you can override the base setting selectively in both directions.

Reply:
I added some tests around what I had already written just to check my assumptions.

I will rework them with the latter set of changes - I wasn't sure if you'd made up your mind.  One argument against multiple tags: you'd have to handle when they're both set with an error or warning or silently picking a winner.  I don't think that's a very realistic concern, but I'd rather not have to worry about it.

I was looking at simplifying things for some cases when you currently need `@foreignSimpleFieldName`, but I found out that apparently this plugin runs too late to deconflict name clashes in that case.  Generally when that tag is in play then the suffix tag is effectively a no-op.

Reply:
>  I assume you mean a string `0` there?
Yep. Smart tags don't have numbers.

> you'd have to handle when they're both set with an error or warning or silently picking a winner

Yeah, I came to this conclusion too, which is why I suggested the `@listSuffix` singular tag.